src/js/constants.js

```diff
@@ -29,6 +29,13 @@ var exports = {
   USER_BLOCK: "user_block",
   USER_COOKIE_BLOCK: "user_cookieblock",
 
+  BLOCKED_ACTIONS: [
```

Comment:
I tried at first to use existing constants, but the scoping wasn't working. If I declare the object like this:

```
  BLOCKED_ACTIONS: {
    BLOCK: true,
    COOKIEBLOCK: true,
    USER_BLOCK: true,
    USER_COOKIE_BLOCK: true
  };
```

then I do `Object.keys(constants.BLOCKED_ACTIONS)` it returns this:

```
["BLOCK", "COOKIEBLOCK", "USER_BLOCK", "USER_COOKIE_BLOCK"]
```

An object makes sense though; shall I swap it to an object with the keys as the current elements in the array?